frontend/src/components/SignupForm.tsx

```diff
@@ -79,45 +86,48 @@ const SignupForm = (props: any) => {
   }, [error, isAuthenticated]);
 
   return (
-    <form>
-      { errorMsg ? <div>{errorMsg}</div> : null}
-      <Input
-        placeholder="Name"
-        id="name"
-        name="name"
-        required
-        className={classes.inputFields}
-        onChange={handleNameChange}
-      />
-      <Input
-        placeholder="Email"
-        type="email"
-        id="email"
-        name="email"
-        required
-        className={classes.inputFields}
-        onChange={handleEmailChange}
-      />
-      <Input
-        placeholder="Password"
-        type="password"
-        id="password"
-        name="password"
-        required
-        className={classes.inputFields}
-        onChange={handlePasswordChange}
-      />
-      <Input
-        placeholder="Retype password"
-        type="password"
-        id="confirmPassword"
-        name="confirmPassword"
-        required
-        className={classes.inputFields}
-        onChange={handleConfirmPasswordChange}
-      />
-      <Button variant="contained" className={classes.loginButton} onClick={handleSubmit}>Sign up</Button>
-    </form>
+    <div>
+      <form>
+        {errorMsg ? <div>{errorMsg}</div> : null}
+        <Input
+          placeholder="Name"
+          id="name"
+          name="name"
+          required
+          className={classes.inputFields}
+          onChange={handleNameChange}
+        />
+        <Input
+          placeholder="Email"
+          type="email"
+          id="email"
+          name="email"
+          required
+          className={classes.inputFields}
+          onChange={handleEmailChange}
+        />
+        <Input
+          placeholder="Password"
+          type="password"
+          id="password"
+          name="password"
+          required
+          className={classes.inputFields}
+          onChange={handlePasswordChange}
+        />
+        <Input
+          placeholder="Retype password"
+          type="password"
+          id="confirmPassword"
+          name="confirmPassword"
+          required
+          className={classes.inputFields}
+          onChange={handleConfirmPasswordChange}
+        />
+        <Button variant="contained" className={classes.loginButton} onClick={handleSubmit}>Sign up</Button>
+      </form>
+      <img src={Books} className={classes.imageBG} alt="books" />
```

Comment:
this shows up in the signup modal. move this to the signup page instead. since the `img` tag won't be here you can probably remove the `div` here too once this change has been made.

Reply:
yee fixed